test/testcafe/spec/SelectAuthenticatorForVerification_spec.js

```diff
@@ -209,7 +210,21 @@ test.requestHooks(mockChallengePassword)('should load select authenticator list'
   // signout link at enroll page
   await t.expect(await selectFactorPage.signoutLinkExists()).ok();
   await t.expect(selectFactorPage.getSignoutLinkText()).eql('Back to sign in');
+});
 
+test.requestHooks(mockChallengePassword)('should load select authenticator list with no sign-out link', async t => {
+  const selectFactorPage = await setup(t);
+  await renderWidget({
+    features: { hideSignOutLinkInMFA: true },
+  });
+  await t.expect(selectFactorPage.getFormTitle()).eql('Verify it\'s you with an authenticator');
+  // signout link is not visible
+  await t.expect(await selectFactorPage.signoutLinkExists()).notOk();
+});
+
+test.requestHooks(mockChallengePassword)('should load select authenticator list', async t => {
```

Comment:
do we need this test? Seems it is just testing the title, and I think we have that already in tests above

Reply:
I think this was a copy + paste error.. it shouldn't be here! Will remove. 😞